src/inputs/input.stories.html

```diff
@@ -0,0 +1,199 @@
+          </svg>
+        </div>
+        <div class="omrs-input-group">
+          <input
+            id="tel2"
+            class="omrs-input-filled omrs-input-danger"
+            type="text"
+            required
+          />
+          <label for="tel2">Tel</label>
+          <span class="omrs-input-helper">
+            Helper Text
+          </span>
+          <svg class="omrs-icon" role="img">
+            <use xlink:href="#omrs-icon-visibility"></use>
+          </svg>
+        </div>
+
+        <div class="omrs-input-group">
+          <input
+            id="email2"
+            class="omrs-input-filled"
+            type="text"
+            required
+            disabled
+          />
+          <label for="email2">Email</label>
+          <svg class="omrs-icon" role="img">
+            <use xlink:href="#omrs-icon-visibility"></use>
+          </svg>
+          <span class="omrs-input-helper">Helper Text</span>
+        </div>
+      </form>
+    </div>
+    <div>
+      <h3 class="title">Outlined Inputs</h3>
+      <form>
+        <div class="omrs-input-group">
+          <input id="name3" class="omrs-input-outlined" type="text" required />
+          <label for="name3">Names</label>
+          <span class="omrs-input-helper">Helper Text</span>
+          <svg class="omrs-icon" role="img">
+            <use xlink:href="#omrs-icon-visibility"></use>
+          </svg>
+        </div>
+        <div class="omrs-input-group">
+          <input
+            id="tel3"
+            class="omrs-input-outlined omrs-input-danger"
+            type="text"
+            required
+          />
+          <label for="tel3">Tel</label>
+          <span class="omrs-input-helper">
+            Helper Text
+          </span>
+          <svg class="omrs-icon" role="img">
+            <use xlink:href="#omrs-icon-visibility"></use>
+          </svg>
+        </div>
+
+        <div class="omrs-input-group">
+          <input
+            id="email3"
+            class="omrs-input-outlined"
+            type="text"
+            required
+            disabled
+          />
+          <label for="email3">Email</label>
+          <svg class="omrs-icon" role="img">
+            <use xlink:href="#omrs-icon-visibility"></use>
+          </svg>
+          <span class="omrs-input-helper">Helper Text</span>
+        </div>
+      </form>
+    </div>
+  </div>
+  <h3>Code example:</h3>
+  <p class="type-demo omrs-bold"><%=inputType%> input</p>
+  <div class="omrs-input-group">
+    <input
+      id="demoInput"
+      class="<%= determineStyle(inputType, isDanger) %>"
+      type="text"
+      required
+    />
+    <label for="demoInput">Names</label>
+    <svg class="omrs-icon" role="img">
+      <use xlink:href="#omrs-icon-visibility"></use>
+    </svg>
+    <span class="omrs-input-helper">Helper Text</span>
+  </div>
+  <div class="stories-example lang-html">
+    <div class="omrs-input-group">
+      <input
+        for="demoInput"
```

Comment:
This should be `id="demoInput"`, right? The `<input>` should have the id and the `<label>` should have the `for` attribute

Reply:
sure